ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+
+Draft
+
+
+## Problem Statement
+
+There is content within SIG-SRE that is being produced that depends upon the defintion of a "persona" as it is related to Managed Services. For example, as a consumer of SLO's, I want to be able to consume the data in a relevant and consistent way in accordance with my "persona". Similarly, there is a RACI chart which has persona as a key datapoint.
+
+Therefore, to assist in the production of this other content, it is necessary to first define the persona. Some examples are (but not limited to): SRE, Engineer, Product Owner, Business, Executive.
+
+
+## Goals
+
+This ADR hopes to resolve the problem statement by identifying and documenting the key personas in a summary format. The output will be a document which will be accessible by other assets.
+
+
+## Non-goals
+
+* This ADR does not aim at capturing every form or type of persona. It is purposefully generalised, utilising the most frequent persona types as experienced within the Red Hat 'Managed Services' context. The reason is that every organisation calls their personas by different names (and they will inevitably have different role descriptions). Nevertheless, this document hopes to capture the main types in a translatable way.
+
+
+## Current Architecture
+
+* There was some pre-existing art in an internal Red Hat document entitled the "Application Services Primer". That was used as a source for this ADR.
+
+
+## Proposed Architecture
+
+The proposed content is as follows:
+
+***
+
+**Personas related to "Managed Services" (WIP)**
+
+**Purpose**
+To underpin many other Site Reliability Engineering-related topics, we must define the group of "personas" describing  the interaction of that person (or group of people) with an environment supported by SREs.
+
+**Service Owner**
+
+Every service needs an "owner". That is, a persona that has the primary interest in the commercial success of that service. The Service Owner takes responsibility for the success of the service and provides the strategic leadership and business context that is needed by engineers, especially during the difficult times. Without this persona, the service would not exist, and thus all other personas rely on this one. They are often referred to as the 'Champions' of this service.
+
+**Product Owner/Manager**
+
+With the strategic guidance of the Service Owner, this persona (also sometimes referred to as the Product Manager) decides on the priority of work (in this context, that may be categorized as 'feature' work or 'reliability work'). They take their inputs from many sources. Reliability, of course, is a significant input, but it is not the only input. There are many other factors which affect their decision-making. Often, they need to balance the inputs from customers and the business, with the engineering resources that are available and these demands can often be conflicting. It is the task of this role to successfully navigate all the demands, providing a path forward.
```

Comment:
I am wondering what's the distinction between 'service' and 'product' in this context, 'Managed Services'.
What is the service and what is the product?
From reading the descriptions, the roles sound like a split between strategic and tactical success concerns. Maybe thats what you're trying to capture here?

Reply:
I am resolving this conversation. If it comes up again, we can fix it post-merge?